types/index.ts

```diff
@@ -1,3 +1,7 @@
+import TypeChildren from '@/types/general/TypeChildren'
```

Comment:
should be done with single line
```suggestion
export type { default as TypeChildren } from '@/types/general/TypeChildren'
```